### ISSUE TITLE
Conditionally running steps / flows based on iOS/android

### ISSUE THREAD

**Issue by matt-dalton** [mobile-dev-inc/maestro#302]
Maybe I missed this in the docs, but is there a way to [conditionally](https://maestro.mobile.dev/advanced/conditions) run steps or flows based on whether we're on iOS on Android? At the moment, we have to set certain steps as optional, which isn't ideal because then they don't fail.

**Comment by matt-dalton** [comment]
That would be fantastic. The only way I can think to do this currently is to run flows based on some kind of UI difference between the platforms - in our case there are barely any.

**Comment by AKJAW** [comment]
Maybe having specific per platform actions could be also helpful when the UI is pretty much the same, but some things have to be accessed differently e.g.:
```
- tapOn:
    ios: 
        text: "moreFollowing"
        rightOf: "Lionel Messi"
    Android:
        rightOf: "Lionel Messi"
        below: "See more"
    # rest is the same
```

**Comment by axelniklasson** [comment]
This is available in 1.27.0 - thanks for the contribution @larryng!
